.github/workflows/issue-closed.yml

```
@@ -0,0 +1,28 @@
+name: issue closed
+
+on:
+  issues:
+    types: [closed]
+
+jobs:
+  issue-closed:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v6
+        with:
+          script: |
```

Hey @pd93,

We have the @task-bot account which is already used for some automation like the website build & publish process at the [gh-pages](https://github.com/go-task/task/tree/gh-pages) branch.

I think we can use it for these scripts as well. Its tokens is set to `GH_PAT`. We need to apply it to the other GA workflow files as well.

Doc: https://github.com/actions/github-script#using-a-separate-github-token

```suggestion
          github-token: ${{secrets.GH_PAT}}
          script: |
```